File: 1351. Count Negative Numbers in a Sorted Matrix/1351.cpp
class Solution {
public:
    int countNegatives(vector<vector<int>>& grid) {
        int count = 0;
        int myLen = grid[0].size();
        for (int i=0; i<grid.size(); i++){
            for (int j=0; j<myLen; j++){
                if (grid[i][j]<0){
                    count += (myLen-j);
                    break;
                }
            }
        }
        return count;
    }
};

File: 1351. Count Negative Numbers in a Sorted Matrix/1351.go
package leetcode

func countNegatives(grid [][]int) int {
    myLen := len(grid[0])
	count := 0
	for _,v := range grid{
		for index, value := range v{
			if value<0{
				count += (myLen-index)
                break
			}
		}
	}
	return count
}